packages/amplication-server/src/core/module/module.service.ts

```diff
@@ -100,6 +145,20 @@ export class ModuleService extends BlockTypeService<
     }
 
     this.validateModuleName(args.data.name);
+
+    const subscription = await this.billingService.getSubscription(
+      user.workspace?.id
+    );
+
+    await this.analytics.trackWithContext({
+      properties: {
+        name: args.data.name,
+        planType: subscription.subscriptionPlan,
+        operation: "rename",
```

Comment:
Is this the only possible update for Module (only renaming)?

Reply:
name and description @mulygottlieb